### PR TITLE
Add a first version of ecology tools list

### PR DESCRIPTION
**Pull request by yvanlebras**
- [ ] check if all non-`mnhn65mo` are already installed
  - [ ] it not review and condafy the non-installed once
- [ ] review the `mnhn65mo` based tools
- [ ] check categories

### CODE REVIEW

File: ecology.yaml [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+install_resolver_dependencies: true
+install_tool_dependencies: false
+
+tools:
+
+  - name: 'collapse_collections'
+    owner: 'nml'
+    tool_panel_section_label: 'Collection Operations'
+
+  - name: dwt_var_perfeature
```
Comment (bgruening) [inline ecology.yaml]: I condafied this one: https://github.com/galaxyproject/tools-devteam/pull/510

File: ecology.yaml [hunk 2]
```diff
@@ -0,0 +1,515 @@
...
+    tool_panel_section_label: Join, Subtract and Group
+
+  # - install_repository_dependencies: true
+  #   install_resolver_dependencies: true
+  #   install_tool_dependencies: false
+  #   name: datamash_wrapper
+  #   owner: agordon
+  #   tool_panel_section_label: Join, Subtract and Group
+
+  - name: suite_datamash
```
Comment (bgruening) [inline ecology.yaml]: This one is doubled here and below.

File: ecology.yaml [hunk 3]
```diff
@@ -0,0 +1,515 @@
...
+  - name: collection_column_join
+    owner: iuc
+    tool_panel_section_label: Join, Subtract and Group
+
+  - name: suite_datamash
+    owner: iuc
+    tool_panel_section_label: Join, Subtract and Group
+
+
+  - name: subtract
```
Comment (bgruening) [inline ecology.yaml]: This is also multiple times included.

File: ecology.yaml [hunk 4]
```diff
@@ -0,0 +1,515 @@
...
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_suite
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+
+
+
+  - name: csv_to_tabular
```
Comment (bgruening) [inline ecology.yaml]: Should this maybe included in Galaxy as datatype converter. So it can be used implicitly?

File: ecology.yaml [hunk 5]
```diff
@@ -0,0 +1,515 @@
...
+
+  - name: ggplot2_histogram
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: tsne
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: best_regression_subsets
```
Comment (bgruening) [inline ecology.yaml]: We need to check all devteam tools if they are either already installed or if we need to condafy them.

File: ecology.yaml (hunk 4 above)
Comment (yvanlebras) [reply ecology.yaml]: In a way we can use it through "Edit attributes/convert format" history dataset option ? Yes for sure it's a good idea... I never add such a kind of "datatype converter", is there something specific to change on the tool form and/or the Galaxy side ?

File: ecology.yaml (hunk 5 above)
Comment (bgruening) [reply ecology.yaml]: They probably are. If you say all of these are already in. Thats fine. I was just worried by the duplication and one of the first tool is a very old one from devteam that I try to migrate to conda in https://github.com/galaxyproject/tools-devteam/pull/510

File: ecology.yaml (hunk 1 above)
Comment (yvanlebras) [reply ecology.yaml]: Thank you!

File: ecology.yaml (hunk 2 above)
Comment (yvanlebras) [reply ecology.yaml]: eagle eye! Thank you! And sorry for that....

File: ecology.yaml (hunk 3 above)
Comment (yvanlebras) [reply ecology.yaml]: Double eagle eye! Thank you! And sorry again for that....

File: ecology.yaml (hunk 4 above)
Comment (bgruening) [reply ecology.yaml]: Look at https://github.com/galaxyproject/galaxy/tree/dev/lib/galaxy/datatypes/converters and here: https://github.com/galaxyproject/galaxy/blob/dev/config/datatypes_conf.xml.sample#L27

I think these are the only places you need to modify.

File: ecology.yaml (hunk 5 above)
Comment (yvanlebras) [reply ecology.yaml]: ok, thanks!

File: ecology.yaml [hunk 6]
```diff
@@ -0,0 +1,497 @@
...
+
+  - name: biosigner
+    owner: ethevenot
+    tool_panel_section_label: Statistics
+
+  - name: lcmsmatching
+    owner: prog
+    tool_panel_section_label: Annotation
+
+  - name: add_input_name_as_column
```
Comment (bgruening) [inline ecology.yaml]: Is this one really needed? Also not updated since 3 years and not condafied.

File: ecology.yaml [hunk 7]
```diff
@@ -0,0 +1,414 @@
...
+
+
+
+
+  - name: csv_to_tabular
+    owner: mnhn65mo
+    tool_panel_section_label: File Conversion
+    
+
+  - name: tabular_to_csv
```
Comment (bgruening) [inline ecology.yaml]: Some for this one I guess.
If this is not up-streamed to Galaxy. I guess IUC is a good place to host them.

File: ecology.yaml [hunk 8]
```diff
@@ -0,0 +1,414 @@
...
+    owner: mnhn65mo
+    tool_panel_section_label: File Conversion
+    
+
+  - name: butterfly_analysis
```
Comment (bgruening) [inline ecology.yaml]: Do you think the tools from `mnhn65mo` are ready for review? Or do you need more time?

Comment (bgruening) [reply ecology.yaml]: I don't think butterfly_analysis is on github?

File: ecology.yaml (hunk 6 above)
Comment (yvanlebras) [reply ecology.yaml]: not needed!

File: ecology.yaml (hunk 8 above)
Comment (yvanlebras) [reply ecology.yaml]: Concerning butterfly_analysis, I think this is the last PR we are waiting from @alanamosse... Tools will be ready for review at the end of the day.

File: ecology.yaml (hunk 7 above)
Comment (yvanlebras) [reply ecology.yaml]: ok I understand. I'm not so familiar with build-in converter, and for sure I need to think more about that, but in a first approach, I think having this as a tool and so the possibility to integrate it on workflows is a good think. So making a PR to IUC will be the best, that's it?

**Comment by bgruening** [comment]
@yvanlebras can you keep this list up to date? Let's aim for the tools that are needed for training ...

**Comment by yvanlebras** [comment]
See up to date list from https://github.com/usegalaxy-eu/usegalaxy-eu-tools/pull/188

**Pull request closed without merging**